x-pack/plugins/ml/public/application/model_management/test_models/test_flyout_wrapper.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC, useState } from 'react';
+
+import {
+  type TestTrainedModelsContextType,
+  TestTrainedModelsContext,
+} from './test_trained_models_context';
+import type { ModelItem } from '../models_list';
+import { TestTrainedModelFlyout } from './test_flyout';
+import { CreatePipelineForModelFlyout } from '../create_pipeline_for_model/create_pipeline_for_model_flyout';
+
+interface Props {
+  model: ModelItem;
+  onClose: () => void;
+}
+export const TestFlyoutWrapper: FC<Props> = ({ model, onClose }) => {
```

Comment:
This is a tricky naming since it contains both the pipeline creation flyout and the test flyout - renamed to `TestModelAndPipelineCreationFlyout` to be explicit. Happy to rename if this doesn't sound clear enough.

869c46cd1958becccee926172373b5e52d125d87